### ISSUE TITLE
typo: setup:config:set  --session-save-redis-sentinel-connect-retires (instead of retries)

### ISSUE THREAD

**Issue by medigeek** [magento/magento2#28802]
### Preconditions (*)

1. 2.3.5-p1
2. 2.4-develop -- in git: https://github.com/magento/magento2/blob/2.4-develop/setup/src/Magento/Setup/Model/ConfigOptionsList/Session.php#L43

`     const INPUT_KEY_SESSION_REDIS_SENTINEL_CONNECT_RETRIES = 'session-save-redis-sentinel-connect-retires';`

### Steps to reproduce (*)

```
$ php bin/magento --help setup:config:set | grep retires
      --session-save-redis-sentinel-connect-retires=SESSION-SAVE-REDIS-SENTINEL-CONNECT-RETIRES  Redis Sentinel connect retries.
```

### Expected result (*)

the command parameter ought to be named "retries" instead of "retires"

### Actual result (*)

```
$ php bin/magento --help setup:config:set | grep retires
      --session-save-redis-sentinel-connect-retires=SESSION-SAVE-REDIS-SENTINEL-CONNECT-RETIRES  Redis Sentinel connect retries.
```

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [x] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

### Note: 

Not 100% sure if it hinders or not the functionality. Didn't test that, on a first glance in the code, I suppose -retires="..." works as expected.

The default sentinel_connect_retries seems to be set correctly and the `constant CONFIG_PATH_SESSION_REDIS_SENTINEL_CONNECT_RETRIES` is correct (without the typo): https://github.com/magento/magento2/blob/2.4-develop/setup/src/Magento/Setup/Model/ConfigOptionsList/Session.php#L67

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Delta 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-35642` were created

**Issue Available**: @engcom-Delta, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @medigeek. Thank you for your report. 
The issue has been fixed in magento/magento2#28816 by @medigeek in 2.4-develop branch
Related commit(s):
 - [2dfa2218604b370840935e65e982eac02f8afa68](https://github.com/magento/magento2/commit/2dfa2218604b370840935e65e982eac02f8afa68)

The fix will be available with the upcoming 2.4.2 release.
